mmv1/products/securitycenterv2/ProjectNotificationConfig.yaml

```diff
@@ -0,0 +1,127 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'ProjectNotificationConfig'
+base_url: projects/{{project}}/locations/{{location}}/notificationConfigs
+self_link: '{{name}}'
+create_url: projects/{{project}}/locations/{{location}}/notificationConfigs?configId={{config_id}}
+update_verb: :PATCH
+update_mask: true
+import_format:
+  - 'projects/{{project}}/locations/{{location}}/notificationConfigs/{{config_id}}'
+description: |
+  A Cloud Security Command Center (Cloud SCC) notification configs. A
+  notification config is a Cloud SCC resource that contains the
+  configuration to send notifications for create/update events of
+  findings, assets and etc.
+  ~> **Note:** In order to use Cloud SCC resources, your organization must be enrolled
+  in [SCC Standard/Premium](https://cloud.google.com/security-command-center/docs/quickstart-security-command-center).
+  Without doing so, you may run into errors during resource creation.
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'Official Documentation': 'https://cloud.google.com/security-command-center/docs'
+  api: 'https://cloud.google.com/security-command-center/docs/reference/rest/v2/projects.locations.notificationConfigs'
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'scc_v2_project_notification_config_basic'
+    primary_resource_id: 'custom_notification_config'
+    vars:
+      topic_name: 'my-topic'
+      config_id: 'my-config'
+    test_env_vars:
+      project: :PROJECT_NAME
+    ignore_read_extra:
+      - 'location'
+      - 'project'
+custom_code: !ruby/object:Provider::Terraform::CustomCode
+  custom_import: templates/terraform/custom_import/self_link_as_name_set_project.go.erb
```

Comment:
this should be what's causing the import issues since it's using the name for the import. The name itself has the project set as the `projectId` rather than the `project`
```
2024/07/25 03:57:36 [DEBUG] [transport] [server-transport 0xc00126b800] loopyWriter exiting with error: transport closed by client 
    vcr_utils.go:152: Step 2/2 error running import: ImportStateVerify attributes not equivalent. Difference is shown below. The - symbol indicates attributes missing after import.
        
          map[string]string{
        - 	"project": "ci-test-project-188019",
        + 	"project": "1067888929963",
          }
```

Reply:
I have used the same thing for the V1 version of the changes in this [PR](https://github.com/GoogleCloudPlatform/magic-modules/pull/11004) and it was working fine and got merged, over there when I had the similar ImportStateVerify error, the reviewer suggested to use the `ignore_read_extra:` in the yaml file to ignore the `project` value, but why is it having issue over here.

Reply:
Looks like all the tests have passed now, if there are no other issues can you please help in merging the PR, before merging can you please squash the commits so that we don't have that commit history.